bolt-sidecar/src/primitives/commitment.rs

```diff
@@ -185,24 +185,24 @@ impl InclusionRequest {
         preconfirmed_gas: u64,
         min_inclusion_profit: u64,
         max_base_fee: u128,
-    ) -> Result<bool, PricingError> {
+    ) -> Result<(bool, u128, u128), PricingError> {
```

Comment:
Are you able to keep the signature like before? `Result<bool, PricingError>` and embed the values in the error instead?

Reply:
Let us know if you get stuck!